### PR TITLE
Create Demo Data Insert Script

### PR DESCRIPTION
**Pull request by ajaygandecha**
This pull request adds a script for organization demo data to be added into the CSXL production database.

The script does not insert any other data, nor does it reset the database.

### CODE REVIEW

File: backend/script/demo_insert.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+from ..test.services.organization import organization_demo_data
+
+__authors__ = ["Kris Jordan"]
+__copyright__ = "Copyright 2023"
+__license__ = "MIT"
+
+if getenv("MODE") != "development":
+    print("This script can only be run in development mode.", file=sys.stderr)
+    print("Add MODE=development to your .env file in workspace's `backend/` directory")
+    exit(1)
```
Comment (ajaygandecha) [inline backend/script/demo_insert.py:21]: Do I need to do anything with this @KrisJordan?

Comment (KrisJordan) [reply backend/script/demo_insert.py:21]: Yes, I believe this will need to be taken out for prod purposes.

**Review by KrisJordan — CHANGES_REQUESTED** [review]
Let's put this in a `backend/script/deployments` directory and name it `organizations_launch`.

Do we still have a demo reset script (for development environment) that clobbers tables and inserts all of the demo data, as well?

**Comment by ajaygandecha** [comment]
> Let's put this in a `backend/script/deployments` directory and name it `organizations_launch`.
> 
> Do we still have a demo reset script (for development environment) that clobbers tables and inserts all of the demo data, as well?

Yes, we do! We still have `demo_reset`. I will rename this script!

**Review by KrisJordan — APPROVED** [review]
lgtm!

**Pull request merged**